workers/generic-worker/schemas/insecure_posix.yml

```diff
@@ -460,6 +459,16 @@ oneOf:
             still includes feature in order for d2g to work with it.
           type: boolean
           default: false
+        containerEngine:
+          title: Container engine
+          description: >-
+            The container engine to use, between `docker` and `podman`.
+            This property is only used during Generic Worker's internal
+            payload translation (d2g).
```

Comment:
I think it might be nicer to say "The container engine to use when executing tasks with Docker Worker formatted task payloads." I don't think you need to list the options in the description, because they are already listed in the enum. I like having d2g in brackets. I think "Generic Worker's internal payload translation" is a little bit too opaque, which is why I think I favor being explicit that it is when Generic Worker executes tasks that were designed for Docker Worker.